# -*- coding: utf-8 -*-

# Form implementation generated from reading ui file 'help.ui'
#
# Created: Tue Feb 21 10:36:06 2012
#      by: PyQt4 UI code generator 4.8.1
#
# WARNING! All changes made in this file will be lost!

from PyQt4 import QtCore, QtGui

try:
    _fromUtf8 = QtCore.QString.fromUtf8
except AttributeError:
    _fromUtf8 = lambda s: s

class Ui_Form(object):
    def setupUi(self, Form):
        Form.setObjectName(_fromUtf8("Form"))
        Form.resize(667, 480)
        self.textBrowser = QtGui.QTextBrowser(Form)
        self.textBrowser.setGeometry(QtCore.QRect(0, 0, 661, 481))
        self.textBrowser.setObjectName(_fromUtf8("textBrowser"))

        self.retranslateUi(Form)
        QtCore.QMetaObject.connectSlotsByName(Form)

    def retranslateUi(self, Form):
        Form.setWindowTitle(QtGui.QApplication.translate("Form", "Form", None, QtGui.QApplication.UnicodeUTF8))
        self.textBrowser.setHtml(QtGui.QApplication.translate("Form", "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.0//EN\" \"http://www.w3.org/TR/REC-html40/strict.dtd\">\n"
"<html><head><meta name=\"qrichtext\" content=\"1\" /><style type=\"text/css\">\n"
"p, li { white-space: pre-wrap; }\n"
"</style></head><body style=\" font-family:\'宋体\'; font-size:9pt; font-weight:400; font-style:normal;\">\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><a name=\"cautions\"></a><span style=\" font-size:20pt; font-weight:600;\">E</span><span style=\" font-size:20pt; font-weight:600;\">XPAR Template Sequence Analysis (ETSeq) </span></p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-size:18pt; font-weight:600;\">Documentation </span></p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><a name=\"SEQUENCE\"></a>Release 0.5.1 </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">Last modified: February 12, 2012 </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">This tool is designed to predict the performance of oligonucleotide templates used in the EXPonential Amplification Reaction (EXPAR). </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-size:18pt; font-weight:600;\">EXPAR - General Assay Principle </span></p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">EXPAR, an isothermal DNA amplification method, amplifies short oligonucleotides, called trigger X, at 55°C. The trigger oligonucleotides that initiate EXPAR can be enzymatically generated from specific sites within the targeted genomic DNA, and therefore represent the analyte. Exponential amplification of trigger X is facilitated by an EXPAR amplification template oligonucleotide provided in excess in the reaction. The EXPAR template contains two copies of the trigger reverse complement X’, separated by the reverse complement of a nicking enzyme recognition site plus a required four base post-cut site spacer. Trigger X primes the template and is extended by a polymerase, which generates a double stranded 5’-GAGTC-3’ on the top strand that is recognized by the nicking enzyme Nt.BstNBI. The nicking enzyme nicks the top strand four bases to the 3’ end of its recognition sequence. This creates another copy of the oligonucleotide trigger X that melts off or is displaced from the amplification template. During EXPAR, templates with similar thermodynamic characteristics often exhibit very different trigger amplification rates, and differ in terms of propensity for non-specific background amplification. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-size:18pt; font-weight:600;\">Purpose of ETSeq</span> </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">ETSeq enables the analysis of a user defined set of EXPAR template sequences to increase the percentage of likely good performers within a subset thereof to be screened experimentally. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">For more information about EXPAR and ETSeq, please see: </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"> <span style=\" background-color:#ffff00;\">&lt;we will insert the reference to the NAR manuscript here&gt;</span> </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">For each entered sequence, the tool returns 7 parameters to the user. Three parameters, related to the thermodynamic behavior of the template, are calculated using the output of UNAFold. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">For more information about Unafold: </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">Markham, N. R. &amp; Zuker, M. (2008) UNAFold: software for nucleic acid folding and hybridization. In Keith, J. M., editor, <span style=\" font-style:italic;\">Bioinformatics, Volume II. Structure, Function and Applications</span>, number 453 in <span style=\" font-style:italic;\">Methods in Molecular Biology</span>, chapter 1, pages 3–31. Humana Press, Totowa, NJ. ISBN 978-1-60327-428-9. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">The other four parameters relate to the sequence dependence of template performance. These four sequence dependence related parameters are generated by two machine learning methods: a position weight matrix based method, as described in <span style=\" background-color:#ffff00;\">&lt;NAR ref&gt;</span>, and a naïve Bayes method, using the Orange Naïve Bayes module. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">For more information about the Orange Naïve Bayes module: </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">Tomaž Curk, Janez Demšar, Qikai Xu, Gregor Leban, Uroš Petrovič, Ivan Bratko, Gad Shaulsky, Blaž Zupan. <span style=\" font-style:italic;\">Microarray data mining with visual programming</span>. Bioinformatics. 2005 Feb 1;21(3):396-8. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">In addition, ETSeq suggests a set of possibly well performing EXPAR templates selected from the user provided set. If no sequence is returned, then the software determined that none of the provided sequences fall within the recommended parameter range. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-size:18pt; font-weight:600;\">Downloading and Installing: </span></p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">ETSeq can be downloaded from: <span style=\" background-color:#ffff00;\">&lt;URL goes here&gt;</span> </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-weight:600;\">Operating system:</span> This software tool has been designed for Windows 7 or XP having at least 512 Mb of RAM and 1Gb of hard drive space. Other operating systems have not yet been tested. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-weight:600;\">Other requirements:</span> This tool integrates the UNAFold perl scripts to predict thermodynamic characteristics. Therefore, users need to install a perl compiler, such as Activeperl, on their computer before using this tool. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">To download Activeperl: </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><a href=\"http://www.activestate.com/activeperl/downloads\"><span style=\" text-decoration: underline; color:#0000ff;\">http://www.activestate.com/activeperl/downloads</span></a> </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-weight:600;\">Installing the program:</span> Unzip the file “ETseq.zip” to a designated folder. To check proper installation, double click the file “run.exe” in your unzipped designated folder. A user interface window will open, which means you have installed the tool successfully. An error message “Perl compiler required” indicates that a Perl compiler is either missing or not properly installed on the computer. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:18pt; font-weight:600;\">Running instruction:</span> </p>\n"
"<ol style=\"margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;\"><li style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">Double click the file “Run.exe” in your ETSeq designated folder - </span>a Graphical User Interface (GUI) window will open. </li>\n"
"<li style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">Copy and paste your template identifiers and sequences into the input area on the ETSeq GUI (see below for formatting requirements)</span> </li>\n"
"<li style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">If desired, change the thermodynamic input parameters (or leave as default)</span> </li>\n"
"<li style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">Click “Submit”</span> </li>\n"
"<li style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">Run time can vary from a few seconds to minutes or longer, based on how many template sequences were submitted</span> </li>\n"
"<li style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">Another dialog window will open when the analysis is completed, click “save”</span> </li>\n"
"<li style=\" margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">Save the result </span>as Excel spreadsheet (.xls format)<span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\"> to a chosen</span> folder. </li></ol>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">  </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-size:18pt; font-weight:600;\">Input Format for EXPAR Template Sequences: </span></p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">The program can accept an unrestricted number of EXPAR template input sequences at one time. However, run times may be long for large input files. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">EXPAR template sequences must be entered in the 5’ to 3’ direction, and must consist of the following elements: (1) the reverse complement of the trigger sequence (at least 10 bases in length), (2) a 4-base post cut site, (3) GACTC, which is the reverse complement of the Nt.BstNBI nicking enzyme recognition site, and (4) a second copy of the reverse complement of the trigger sequence. The EXPAR template must have exactly one GACTC in its sequence. Only exact nucleotides (A, C, G, or T) are allowed. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">Each EXPAR sequence requires a unique user defined sequence identifier (no blank spaces in user identifier). If multiple EXPAR template sequences have the same sequence identifier, then all but the first will be deleted by the program. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">Enter the EXPAR template sequences to be analyzed into the input field. There are two acceptable input formats: </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">1. EXPAR template sequences plus sequence identifiers can be entered in fasta format (see <a href=\"http://blast.ncbi.nlm.nih.gov/blastcgihelp.shtml\"><span style=\" text-decoration: underline; color:#0000ff;\">http://blast.ncbi.nlm.nih.gov/blastcgihelp.shtml</span></a> for a description of the FASTA format). Generate this file using another suitable computational tool, or using a simple text editor such as the Windows notepad accessory, then copy and paste into the input area. The input data would look as follows: </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">&gt;seq1 </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">CCTACGACTGAACAGACTCTCCTACGACTG </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">&gt;seq2 </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">CCTACGACTTAACAGACTCTCCTACGACTT </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">2. Generate an Excel file (or a tab delimited text file) with sequence identifiers in column 1, and the sequences in column 2. Copy and paste this data (without header rows) into the input area, resulting in a tab delimited format such as the following: </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">seq1  CCTACGACTGAACAGACTCTCCTACGACTG </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">seq2  CCTACGACTTAACAGACTCTCCTACGACTT </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-size:18pt; font-weight:600;\">Other Input Parameters </span></p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><a name=\"PRIMER_SEQUENCE_ID\"></a>Minimal trigger-template Tm </p>\n"
"<p style=\" margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">The Minimal melting temperature allowed for trigger- template duplex. Default: 40</span><span style=\" font-family:\'Symbol\'; font-size:12pt;\">°</span><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">C</span> </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">Maximal trigger-template Tm </p>\n"
"<p style=\" margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">The Maximal melting temperature allowed for trigger- template duplex. Default: 60</span><span style=\" font-family:\'Symbol\'; font-size:12pt;\">°</span><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">C</span> </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">Maximal template-template Tm </p>\n"
"<p style=\" margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">The Maximal melting temperature allowed for template- template duplex. Default: 25</span><span style=\" font-family:\'Symbol\'; font-size:12pt;\">°</span><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">C</span> </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">Maximal template self-bonds </p>\n"
"<p style=\" margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">The Maximal number of hybridization bonds allowed for template- template hybridization. Default: 10</span> </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-size:18pt; font-weight:600;\">Output Format </span></p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">An excel file containing three sheets will be generated by clicking the “save” button. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">The sheet labeled “Good and desired thermo” contains all the templates that are predicted to be well performing by both PWM based method and naïve bayes method, and predicted to be have the desired thermodynamic characteristics as well. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">“desired thermo” is for all the templates which are predicted to be have the desired thermodynamic characteristics. </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">“all submitted templates” is for all the templates which are submitted to the tool. The meaning of the header in the sheets are listed below: </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-size:18pt; font-weight:600;\">Output Parameters </span></p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><a name=\"disclaimer\"></a>Each spreadsheet within the Excel output file, contains 9 columns with the following headers </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt; font-weight:600;\">name</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">The sequence ID</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt; font-weight:600;\">template_seq</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">The EXPAR template sequence</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt; font-weight:600;\">bayes_class</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">This is the template performance class predicted by the Naïve Bayes method. “Good” indicates fast amplification of trigger X, and decent temporal separation between specific amplification in the presence of trigger X, and non-specific amplification in the absence of trigger X. “Bad” indicates slow amplification and / or poor temporal separation between specific and non-specific amplification. The bayes_class is based on a Naïve Bayes classifier, obtained using the Orange Naïve Bayes module, that uses significant position motifs derived from previously characterized EXPAR template sequences. This Naïve Bayes classifier and the significant position motifs are embedded in the program.</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt; font-weight:600;\">pwm_class</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">This is the template performance class predicted by the position weight matrix based method. “Good” indicates fast amplification of trigger X, and decent temporal separation between specific amplification in the presence of trigger X, and non-specific amplification in the absence of trigger X. “Bad” indicates slow amplification and / or poor temporal separation between specific and non-specific amplification. The pwm_class is based on two PWMs and a support vector machine based classifier, which are embedded in the program and have been derived from previously characterized EXPAR template sequences.</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt; font-weight:600;\">p90 score</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">This score is generated by the position weight matrix base method. A lower p90 score indicate faster amplification and vice versa. The P90 score is based on a PWM embedded in the program, which is derived from previously characterized EXPAR template sequences.</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt; font-weight:600;\">diff score</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">This score is generated by the position weight matrix base method. A lower diff score indicates decent temporal separation between specific and non-specific amplification. The diff score is based on a PWM embedded in the program, which is derived from previously characterized EXPAR template sequences.</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt; font-weight:600;\">tri-temp Tm</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">Trigger- template duplex melting temperature, assuming that the template oligonucleotide is present in excess in the reaction. This parameter is calculated from the UNAFold output values for </span><span style=\" font-family:\'Symbol\'; font-size:12pt;\">D</span><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">H and </span><span style=\" font-family:\'Symbol\'; font-size:12pt;\">D</span><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">S for trigger template hybridization.</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt; font-weight:600;\">temp-temp Tm</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">The template- template duplex melting temperature. In this case, the two strands are present in equal concentrations; therefore the output value for the template template melting temperature is used directly.</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt; font-weight:600;\">temp bonds</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">The number of template- template hybridization bonds.</span> </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-size:18pt; font-weight:600;\">Copyright Notice and Disclaimer</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">Copyright (C) 2012 - Keck Graduate Institute</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or any later version.</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.</span> </p>\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">A copy of the GNU General Public License is included in the download file of this program.  If not, the licensing agreement can be obtained at </span><a href=\"http://www.gnu.org/licenses/\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt; text-decoration: underline; color:#000099;\">http://www.gnu.org/licenses/</span></a><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">.</span> </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-size:18pt; font-weight:600;\">Citing ETSeq </span></p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">We request that use of this software be cited in publications as: </p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" background-color:#ffff00;\">&lt;we will insert the reference to the NAR manuscript here&gt;</span> </p>\n"
"<p style=\" margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\">Source code available at </span><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt; background-color:#ffff00;\">&lt;we will insert the final url here&gt;</span><span style=\" font-family:\'Times New Roman,serif\'; font-size:12pt;\"> </span></p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\"><span style=\" font-size:18pt; font-weight:600;\">Acknowledgments </span></p>\n"
"<p style=\" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">The development of ETSeq was funded by the National Institutes of Health through award R01AI076247, plus an ARRA supplement to this award. This work was funded in part by the National Science Foundation\'s Frontiers in Integrative Biological Research grant FIBR-0527023, and NSF\'s BEACON Center for the Study of Evolution in Action under contract No. DBI-0939454. </p></body></html>", None, QtGui.QApplication.UnicodeUTF8))

